src/mon/FSCommands.cc

```diff
@@ -108,6 +108,19 @@ class FailHandler : public FileSystemCommandHandler
       return -ENOENT;
     }
 
+  bool confirm = false;
+  cmd_getval(cmdmap, "yes_i_really_mean_it", confirm);
+  if (!confirm &&
+      mon->mdsmon()->has_health_warnings({
+	MDS_HEALTH_TRIM, MDS_HEALTH_CACHE_OVERSIZED})) {
```

Comment:
What about other health warnings?

```
enum mds_metric_t {
  MDS_HEALTH_NULL = 0,
  MDS_HEALTH_TRIM,
  MDS_HEALTH_CLIENT_RECALL,
  MDS_HEALTH_CLIENT_LATE_RELEASE,
  MDS_HEALTH_CLIENT_RECALL_MANY,
  MDS_HEALTH_CLIENT_LATE_RELEASE_MANY,
  MDS_HEALTH_CLIENT_OLDEST_TID,
  MDS_HEALTH_CLIENT_OLDEST_TID_MANY,
  MDS_HEALTH_DAMAGE,
  MDS_HEALTH_READ_ONLY,
  MDS_HEALTH_SLOW_REQUEST,
  MDS_HEALTH_CACHE_OVERSIZED,
  MDS_HEALTH_SLOW_METADATA_IO,
  MDS_HEALTH_CLIENTS_LAGGY,
  MDS_HEALTH_CLIENTS_LAGGY_MANY,
  MDS_HEALTH_DUMMY, // not a real health warning, for testing
};
```

Esp, `MDS_HEALTH_SLOW_REQUEST`  - where the MDS could probably running close to its limits.

Reply:
I thought about it but didn't include them because the ticket doesn't mention them.

Reply:
I've added a comment on ticket about this.

Reply:
Maybe we should add `MDS_HEALTH_SLOW_METADATA_IO`, which I have been seen several times because of the overloading of Rados.

And IMO it's also worth for the `MDS_HEALTH_SLOW_REQUEST`.